core/app/[locale]/layout.tsx

```diff
@@ -16,9 +16,16 @@ import { Notifications } from '../notifications';
 import { Providers } from '../providers';
 
 const inter = Inter({
+  display: 'swap',
   subsets: ['latin'],
+  variable: '--font-family-body',
+});
+
+const dm_serif_text = DM_Serif_Text({
   display: 'swap',
-  variable: '--font-inter',
+  subsets: ['latin'],
+  weight: '400',
+  variable: '--font-family-heading',
 });
```

Comment:
Looks like we didn't bring in `--font-family-mono`

Reply:
I pulled this in from the latest soul docs. Will add, but we need to fix in the docs.